scripts/shared/deploy.sh

```diff
@@ -6,13 +6,15 @@ source ${SCRIPTS_DIR}/lib/shflags
 DEFINE_string 'cluster_settings' '' "Settings file to customize cluster deployments"
 DEFINE_string 'deploytool' 'operator' 'Tool to use for deploying (operator/helm)'
 DEFINE_string 'globalnet' 'false' "Deploy with operlapping CIDRs (set to 'true' to enable)"
+DEFINE_string 'lighthouse' 'false' "Deploy with serice discovery enabled (set to 'true' to enable)"
```

Comment:
I'd rather we switch to boolean type flags for boolean flags, you can see an example here:
https://github.com/submariner-io/shipyard/blob/master/scripts/shared/build_image.sh#L16
https://github.com/submariner-io/shipyard/blob/master/scripts/shared/build_image.sh#L24